core/connection/session.go

```diff
@@ -25,14 +25,43 @@ import (
 	"github.com/mysteriumnetwork/node/session"
 )
 
+// SessionStatus represents list of possible session statuses
+type SessionStatus int
+
+const (
+	// SessionStatusNew means that newly created session object is written to storage
+	SessionStatusNew = SessionStatus(0)
+	// SessionStatusCompleted means that session object is updated on connection disconnect event
+	SessionStatusCompleted = SessionStatus(1)
+)
+
 // Session holds structure for saving session history
 type Session struct {
 	SessionID       session.ID `storm:"id"`
 	ProviderID      identity.Identity
 	ServiceType     string
 	ProviderCountry string
 	TimeStarted     time.Time
```

Comment:
Maybe we can call it just `Started` and `Finished`, it has time `time.Time` no need `Time` word at the name of the field.

Reply:
removed word `Time`